packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/petrinet/Ontology.java

```diff
@@ -0,0 +1,54 @@
+package software.uncharted.terarium.hmiserver.models.petrinet;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import software.uncharted.terarium.hmiserver.proxies.mira.MiraProxy;
+
+import lombok.Data;
+
+import javax.ws.rs.core.Response;
+import java.io.Serializable;
+import java.net.URL;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static io.smallrye.jwt.config.ConfigLogging.log;
+
+@Data
+public class Ontology implements Serializable {
+	private final String name;
+	private final String curie;
+	private String title;
+	private String description;
+	private URL url;
+
+	@RestClient
+	MiraProxy proxy;
+
+	public Ontology (String input) {
+		final Matcher matcher = Pattern.compile("\\(\\'(.+?)\\', \\'(.+?)\\'\\)").matcher(input);
+		final boolean result = matcher.find();
+		int i = matcher.groupCount();
+		if (result && i > 0) {
+			this.name = matcher.group(1);
+			this.curie = matcher.group(2);
+		} else {
+			this.name = this.curie = null;
+		}
+
+		if (this.curie == null) return;
+
+		try {
+			final Response entity = proxy.getEntity(this.curie);
```

Comment:
@dvince2 or @dgauldie, I'm unable to call the _MiraProxy_ from within the _Ontology_ model. 
Any clue? I can't find any documentation about proxies on _Quarkus_ website.

Reply:
You may be missing the `@Inject` annotation on that